docs/en/connector-v2/sink/Jdbc.md

```diff
@@ -201,12 +201,13 @@ CDC(Change data capture) event
 ```
 sink {
     jdbc {
-        url = "jdbc:mysql://localhost:3306/test"
+        url = "jdbc:mysql://localhost:3306"
         driver = "com.mysql.cj.jdbc.Driver"
         user = "root"
         password = "123456"
         
-        table = sink_table
+        database = "sink_database"
+        table = "sink_table"
         primary_keys = ["key1", "key2", ...]
```

Comment:
What's the advantage of this change?

Reply:
The parameters of the dev branch connector have been changed, but not in the doc examples.These two changes were discovered when I was using the dev branch of SeaTunnel.

Reply:
> What's the advantage of this change?

The parameters of the dev branch connector have been changed, but not in the doc examples.These two changes were discovered when I was using the dev branch of SeaTunnel.